pkg/vmprovider/providers/vsphere2/vmlifecycle/bootstrap_sysprep.go

```diff
@@ -26,7 +25,7 @@ func BootstrapSysPrep(
 	var data string
 	key := "unattend"
 
-	if equality.Semantic.DeepEqual(sysPrepSpec.Sysprep, sysprep.Sysprep{}) {
+	if !reflect.ValueOf(sysPrepSpec.RawSysprep).IsZero() {
```

Comment:
Why is this not just checking is not nil?

Reply:
I see what happened. I expected you to make `RawCloudConfig` and `RawSysprep` by address as well.

Reply:
Sure. They're all pointers now.